examples/03_connectivity/plot_seed_to_voxel_correlation.py

```diff
@@ -172,7 +172,7 @@
 # interest.
 from nilearn import plotting
 
-display = plotting.plot_stat_map(seed_based_correlation_img, threshold=0.3,
+display = plotting.plot_stat_map(seed_based_correlations_fisher_z, threshold=0.3,
```

Comment:
On my own data, a threshold of 0.1 seems more appropriate, and gives me really clear-cut examples looking nothing at all like chance (see below).

I am using basically [the code from the example](https://github.com/IBT-FMI/SAMRI/blob/master/samri/pipelines/fc.py#L167)

Is it possible that the computation is incomplete?

![figure_1](https://user-images.githubusercontent.com/950524/27140037-d8d5c898-5124-11e7-9951-a42d99679565.png)